baseband/vdif/tests/test_vdif.py

```diff
@@ -110,6 +111,79 @@ def test_header(self):
         header5.time = header.time + 1.*u.s - 0.01/header.framerate
         assert abs(header5.time - header.time - 1.*u.s) < 1.*u.ns
         assert header5['frame_nr'] == header['frame_nr']
+        # Check requesting non-existent EDV returns VDIFBaseHeader instance
+        header6 = vdif.header.VDIFHeader.fromvalues(edv=100)
+        assert isinstance(header6, vdif.header.VDIFBaseHeader)
+        assert header6['edv'] == 100
+
+    def test_custom_header(self):
+        # Custom header with an EDV that already exists
+        with pytest.raises(ValueError):
+            class VDIFHeaderY(vdif.header.VDIFBaseHeader):
+                _edv = 4
+        # Custom header that has neglected to override _edv
+        with pytest.raises(ValueError):
+            class VDIFHeaderZ(vdif.header.VDIFBaseHeader):
+                pass
+
+        # Working header with nonsense data in the last two words.
+        class VDIFHeaderX(vdif.header.VDIFSampleRateHeader):
+            _edv = 0x1a
```

Comment:
Surely `HeaderX` should have `edv=0x58`? (if you don't see why, try `print('\x58')` on the command line ;-)